fairscale/utils/testing_memory.py

```diff
@@ -11,6 +11,8 @@
 import torch
 
 
+# FIXME: write a test for this.
```

Comment:
Sure. It is a test for a test function though. :-)

This is like "it-works" method for getting the test done. By no means the best. Some times, "perfect is an enemy of good". :-)

Reply:
always write tests for your test functions. you want to _trust_ your test functions, particularly when they do something clever like this :)

Reply:
Very true